### PR TITLE
Add Tests to QA Repo

### PR DESCRIPTION
**Pull request by rajat-soda**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rajat-soda** [comment]
@kumarashit @skdwriting : Please merge test cases to QA repo

**Review by kumarashit — DISMISSED** [review]
LGTM

**Comment by skdwriting** [comment]
1) Testcase ID to be filled. 
2) PLease ensure the test case is complete, else please do not add for merging. Kindly review the test cases once

**Review by skdwriting — COMMENTED** [review]
I think this format is not scalable and hard to update!

**Comment by skdwriting** [comment]
I see errors at the end of certain files, also not consistent format across.?
please check

**Comment by skdwriting** [comment]
Are these test cases are all valid and clean?
Also, how is the edit option? too many commas to handle multiple columns right?
Text wrap?

**Comment by rajat-soda** [comment]
steps to edit the files is provided in the Readme file .
Its not recommended to edit csv files directly, rather open it in Excel / G. Sheets and check it back as CSV.
Text wrap not supported in csv.

**Comment by skdwriting** [comment]
The test cases can be organized as project specific and overall..?
multicloud
delfin
....
common
?

File: testcases/Dock/Hotpot.csv [hunk 1]
```diff
@@ -0,0 +1,223 @@
+Feature Scenario,Description,TestcaseId,Testcase Precondition,Testcase steps,Expected Result
```
Comment (kumarashit) [inline testcases/Dock/Hotpot.csv:1]: Hotpot name is confusing

File: testcases/Dock/NBP_CSI.csv [hunk 2]
```diff
@@ -0,0 +1,320 @@
+Feature Scenario,Testcase Title,TestcaseId,Testcase Precondition,Testcase steps,Expected Result
```
Comment (kumarashit) [inline testcases/Dock/NBP_CSI.csv:1]: this is specific to nbp not dock

**Review by kumarashit — DISMISSED** [review]
LGTM

File: testcases/Dock/NBP_CSI.csv (hunk 2 above)
Comment (rajat-soda) [reply testcases/Dock/NBP_CSI.csv]: Fixed

**Comment by rajat-soda** [comment]
> The test cases can be organized as project specific and overall..?
> multicloud
> delfin
> ....
> common
> ?

Done

**Review by kumarashit — APPROVED** [review]
LGTM

**Review by skdwriting — APPROVED** [review]
LGTM
(Needs refinement on the organization)

**Pull request merged**